src/databricks/labs/ucx/install.py

```diff
@@ -112,68 +111,74 @@ def extract_major_minor(version_string):
     return None
 
 
-class WorkspaceInstaller:
+class WorkspaceInstaller(WorkspaceContext):
+
+    @cached_property
```

Comment:
nit: it's VERY uncommon for methods to be declared before the constructor. can you move them below constructor?..